docs/en/start/api_plugin/generate.md

```diff
@@ -0,0 +1,50 @@
+# Generate API plugin automatically
+
+Writing the API plugin yourself is boring. You can use code generator to generate all the code.
+
+Let's say you want to add a `PublishTransactionalMessage` method to existing pubsub API. You write a new proto file [cmd/layotto_multiple_api/advanced_queue/advanced_queue.proto](cmd/layotto_multiple_api/advanced_queue/advanced_queue.proto) :
```

Comment:
This caused deadlink job failed, how about using the full url or just remove this link and turn it into a plain text.

Reply:
fixed. Thanks